src/RuleFilter/Bridge/LaravelSetProvider.php

```diff
@@ -8,13 +8,40 @@
 use Rector\Set\Contract\SetProviderInterface;
 use Rector\Set\ValueObject\Set;
 use RectorLaravel\Set\LaravelSetList;
+use RectorLaravel\Set\Packages\Livewire\LivewireSetList;
 
 /**
- * @todo should be part of the Laravel package, move later
  * @see https://github.com/driftingly/rector-laravel/tree/main/config/sets
  */
 final class LaravelSetProvider implements SetProviderInterface
```

Comment:
Awesome!

While we're at it, this `LaravelSetProvider` class should be part of https://github.com/driftingly/rector-laravel/ code.

Could you add it there, so we can just re-use the dependency?

Reply:
Yes, of course. My only question is about `GroupName::LARAVEL`, will it be fine if the value of `laravel` is just used as the class is in the getrector repository?